packages/bitcore-node/src/modules/ethereum/p2p/EthVerificationPeer.ts

```diff
@@ -57,4 +68,150 @@ export class EthVerificationPeer extends EthP2pWorker implements IVerificationPe
   async getBlockForNumber(blockNum: number) {
     return this.getBlock(blockNum);
   }
+
+  async validateDataForBlock(blockNum: number, tipHeight: number, log = false) {
+    let success = true;
```

Comment:
duplicated from https://github.com/bitpay/bitcore/pull/2587/files#diff-93e5d9954cc14dd759f4d480c52effb3L48

Removed logic for coins